packages/react-components/src/templates/TeamCreateOutputPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ResearchOutput } from '@asap-hub/model';
+
+import { TeamCreateOutputHeader, TeamCreateOutputForm } from '../organisms';
+
+type TeamCreateOutputPageProps = {
```

Comment:
Ideally we compose props of child components using : 
`type TeamCreateOutputHeaderProps = ComponentProps<typeof ChildComponent>` this allows us to avoid redefining them and having to change them every time a new prop is added

Reply:
Fabulous to know! Done!